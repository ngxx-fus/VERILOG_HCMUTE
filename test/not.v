module not_gate (input in, output out);
    wire in;
    reg out;
    // assign notA = ~A;
    always @(in) begin
        out = ~in;
    end
endmodule
