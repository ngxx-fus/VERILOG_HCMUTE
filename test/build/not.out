#! /c/Source/iverilog-install/bin/vvp
:ivl_version "12.0 (devel)" "(s20150603-1539-g2693dd32b)";
:ivl_delay_selection "TYPICAL";
:vpi_time_precision + 0;
:vpi_module "C:\iverilog\lib\ivl\system.vpi";
:vpi_module "C:\iverilog\lib\ivl\vhdl_sys.vpi";
:vpi_module "C:\iverilog\lib\ivl\vhdl_textio.vpi";
:vpi_module "C:\iverilog\lib\ivl\v2005_math.vpi";
:vpi_module "C:\iverilog\lib\ivl\va_math.vpi";
S_000001cfc2dd2d00 .scope module, "not_gate" "not_gate" 2 1;
 .timescale 0 0;
    .port_info 0 /INPUT 1 "in";
    .port_info 1 /OUTPUT 1 "out";
o000001cfc2e06ef8 .functor BUFZ 1, C4<z>; HiZ drive
v000001cfc2dd3580_0 .net "in", 0 0, o000001cfc2e06ef8;  0 drivers
v000001cfc2dd2e90_0 .var "out", 0 0;
E_000001cfc2e056c0 .event anyedge, v000001cfc2dd3580_0;
    .scope S_000001cfc2dd2d00;
T_0 ;
    %wait E_000001cfc2e056c0;
    %load/vec4 v000001cfc2dd3580_0;
    %inv;
    %store/vec4 v000001cfc2dd2e90_0, 0, 1;
    %jmp T_0;
    .thread T_0, $push;
# The file index is used to find the file name in the following table.
:file_names 3;
    "N/A";
    "<interactive>";
    "not.v";
